doc/howto/cluster_groups.md

```diff
@@ -27,12 +27,11 @@ To assign `server1` to the `gpu` group and also keep it in the `default` group,
 
     lxc cluster group assign server1 default,gpu
 
-To add a cluster member to a specific group, use `lxc cluster group add` command.
-This command adds a cluster member to the specified group.
+To add a cluster member to a specific group without removing it from other groups, use the `lxc cluster group add` command.
 
-For example, to add `cluster:node1` to the `newgroup` group , use the following command:
+For example, to add `server1` to the `gpu` group and also keep it in the `default` group, use the following command:
```

Comment:
Ah I see you've made the requested changes as new commit, but please can you rebase and squash/edit your earlier commit so that we dont see the evolution of the particular change in the final PR.